src/NexusMods.App.UI/Pages/MyLoadouts/MyLoadoutsView.axaml

```diff
@@ -0,0 +1,70 @@
+<reactiveUi:ReactiveUserControl x:TypeArguments="myLoadouts:IMyLoadoutsViewModel" xmlns="https://github.com/avaloniaui"
+                                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                                xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                                xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                                xmlns:reactiveUi="http://reactiveui.net"
+                                xmlns:myLoadouts="clr-namespace:NexusMods.App.UI.Pages.MyLoadouts"
+                                xmlns:panels="clr-namespace:Avalonia.Labs.Panels;assembly=Avalonia.Labs.Panels"
+                                xmlns:gameLoadoutsSectionEntry="clr-namespace:NexusMods.App.UI.Pages.MyLoadouts.GameLoadoutsSectionEntry"
+                                xmlns:ui="clr-namespace:NexusMods.App.UI"
+                                mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+                                x:Class="NexusMods.App.UI.Pages.MyLoadouts.MyLoadoutsView">
+    <Design.DataContext>
+        <myLoadouts:MyLoadoutsDesignViewModel />
+    </Design.DataContext>
+
+
+    <ScrollViewer>
+        <panels:FlexPanel x:Name="MyLoadoutsOuterFlexPanel">
+
+            <TextBlock x:Name="PageDescriptionTextBlock"
+                       Text="Create game Loadouts which contain their own individual mod preferences and settings. Switch between Loadouts and apply the one you want to play." />
+
+            <ItemsControl x:Name="GameSectionsItemsControl">
+                <ItemsControl.ItemsPanel>
+                    <ItemsPanelTemplate>
+                        <panels:FlexPanel x:Name="GameSectionsItemsFlexPanel" />
+                    </ItemsPanelTemplate>
+                </ItemsControl.ItemsPanel>
+
+                <ItemsControl.ItemTemplate>
+                    <DataTemplate DataType="gameLoadoutsSectionEntry:IGameLoadoutsSectionEntryViewModel">
+                        <Border x:Name="GameSectionBorder">
+                            <panels:FlexPanel x:Name="GameSectionFlexPanel">
+
+                                <panels:FlexPanel x:Name="HeadingSectionFlexPanel">
+                                    <TextBlock x:Name="GameHeadingTextBlock"
+                                               Text="{CompiledBinding HeadingText}" />
+
+                                    <panels:FlexPanel x:Name="SupportedDetails">
+                                        <TextBlock x:Name="LoadoutsContainTextBlock"
+                                                   Text="Loadouts contain unique: Mod lists, Mod preferences, Configs." />
+                                        <TextBlock x:Name="LoadoutsShareTextBlock"
+                                                   Text="Loadouts share: Mod library, Saved games." />
```

Comment:
From figma:
![image](https://github.com/user-attachments/assets/0f36cd3c-5718-4c92-9a67-4da9b3030c4a)

Reply:
@captainsandypants is that just some placeholder text or do you really want this in the App? The text doesn't make a lot of sense to me and just seems odd.

Reply:
@erri120 I want that in the app for people who don't know what they are. Although I'm open to suggestions on changing the wording. This will probably come out of user testing organically though.

Reply:
> Loadouts contain unique: Mod lists, Mod preferences, Configs.
> Loadouts share: Mod library, Saved games.

That's the text I'm referring to. As a user, I don't understand what this is supposed to tell me. As a maintainer, I don't know whether this is hardcoded static text or generated based on what is inside the Loadout.

Reply:
This is telling you what Loadouts support for that game. We need to communicate if one game we can give you Loadout specific saved games but in another we can't.

Reply:
Where does this information come from? This is a big can of worms that we probably shouldn't open with this PR and that should go into a separate issue.

Reply:
So hardcoded for now.
We don't even know if there is going to be any game for which we will have a different behavior in the future.

Reply:
It comes from us and what we know Loadouts support for that game. It can be hard coded.

Reply:
It will be hard coded to be the same for all games, it will not be a per game decision. There is no game that behaves differently and we had no discussions to indicate that different games should have different behaviors.

Reply:
I feel like this text shouldn't be here at all. Whatever we put here is going to be incomplete, possibly outdated, and confusing. Can we remove the text for now until we know what we want from this, if anything at all?

Reply:
This has been discussed and we decided that there's a high probability that loadouts will support more/less in different games. If we weren't transparent about this then we would have to restrict our features to the minimum of what's possible. This was removed before that discussion and then adding back in after. Without this text, how do I know if I have unique saved games in each loadout? Or configs?

Reply:
We don't have any games currently where we have different behavior, so no concrete idea how an implementation of that would look like. Would this be a setting? How would it work if changed, would it be a setting per profile?

In any case adding customization or a way to detect the outcome of any customization is outside the scope of this PR.
I'm ok with leaving the text for now.

Reply:
Even if we have no games with different behaviour currently, we still need to explain what is shared and what isn't shared between loadouts. So it either goes here, and future proofs us to a change we know will likely come, or it goes in the intro header bit making the explanation longer and more complicated.

Reply:
I'm fine with leaving it here, it will require to be updated in the future if we ever add a game were we want to have something different.

Reply:
Update based on agreed improvement to the header text:

> Supports individual: Mods and mod configurations.
> Shares: Library and saved games.

Reply:
"Supports" wouldn't be ideal, I think, as it suggests something that it is capable of doing rather than something that it always does. And loadouts will always have separate mods and mod configs.

I would suggest:
>Loadouts contain unique: Mods and mod configurations.
>Loadouts share: Mod Library and saved games.

If loadouts will indeed have different sharing behaviors in other games, we can revisit the language here at that time.
The UI might need to change more drastically than just some text anyways (checkboxes, or more options or buttons).

Reply:
Sounds good